pkg/skr/awsnfsvolumebackup/state.go

```diff
@@ -87,6 +87,12 @@ func (s *State) GetRecoveryPointArn() string {
 	return arn
 }
 
+func (s *State) GetBackupRoleArn() string {
+	arn := fmt.Sprintf("arn:aws:iam::%s:role/service-role/AWSBackupDefaultServiceRole",
+		s.Scope().Spec.Scope.Aws.AccountId)
```

Comment:
Consider passing the role name via config (like its done for vpc peering).
It will make us more flexible in case we decide to stop using the default role.

Reply:
Done